interpreter/cling/lib/Utils/Validation.cpp

```diff
@@ -0,0 +1,69 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Vassil Vassilev <vasil.georgiev.vasilev@cern.ch>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#include "cling/Utils/Validation.h"
+
+#include <assert.h>
+#include <errno.h>
+#ifdef LLVM_ON_WIN32
+# include <Windows.h>
+#else
+#include <fcntl.h>
+# include <unistd.h>
+#endif
+
+
+
+namespace cling {
+  namespace utils {
+
+    static int getNullDevFileDescriptor() {
+#ifdef LLVM_ON_WIN32
```

Comment:
On WIN32 this function shouldn't get called - can you ifndef the whole function declaration? Else we have an unused static function which can / should trigger a warning.